README.md

```diff
@@ -78,6 +78,13 @@ jupyter nbextension enable --py --sys-prefix widgetsnbextension  # can be skippe
 conda install -c conda-forge ipywidgets
 ```
 
+- With JupyterLab (note that it requires nodejs to be installed and that the package name is different):
```

Comment:
Can you add that this JupyterLab step is in addition to the pip or conda step above?